### ISSUE TITLE
Support mixins method

### ISSUE THREAD

**Issue by innocenzi** [Wulfheart/laravel-actions-ide-helper#12]
https://github.com/barryvdh/laravel-ide-helper supports the `-M` flag, that will create class names like `IdeHelperAircraftType` instead of `AircraftType`. 

It would be super nice for this package to support that as well, because right now, following the symbols for an action class lead to the IDE helper file instead of the actual action (on VSCode at least, I assume PHPStorms handles that correctly).

Basically, this flag would:
- Add `@mixin IdeHelperSomeAction` to the `SomeAction.php` action
- Create the same helper file but with `IdeHelperSomeAction`-type class names

**Comment by Wulfheart** [comment]
Nice idea. Feel free to open a PR. 👍🏻

**Comment by innocenzi** [comment]
Cool! I took a spin at it and I easily managed to generate the correct class names, but I don't know yet how to update the action files. I'll see when I have a bit more time.

**Comment by Wulfheart** [comment]
Take a look at BuildIdeHelper. Maybe this gives you a little guidance.

**Comment by vpratfr** [comment]
+1 on this one as this would make the resulting code more PHPStan friendly :)

**Comment by Wulfheart** [comment]
Ok. Time is scarce right now. A PR would definitely help.

**Comment by innocenzi** [comment]
FYI I'm no longer using this package so I will not be able to make this PR.

**Comment by Wulfheart** [comment]
To be honest this package could be considered unmaintained as well. It would be nice to find a maintainer. Unfortunately, I rarely use Laravel Actions anymore.

**Comment by vpratfr** [comment]
@Wulfheart I don't have much time either for new features, but I would be ok to review some PR and make new releases. We are using the package in a recent project at work, hence I should be monitoring at least Laravel updates.

Let me know if you would like some help to get small PRs such as L10 compat merged in new versions quickly.

I authored and maintain a couple of Laravel packages too (https://github.com/marvinlabs/laravel-luhn for instance)

**Comment by Wulfheart** [comment]
This project is in [YOLO-mode](https://github.com/Wulfheart/laravel-actions-ide-helper?tab=readme-ov-file#yolo-ware) now. Please feel free to re-open if you think it still applies.
